### PR TITLE
DOC: Analysis Tutorial

### PR DESCRIPTION
**Pull request by mschoettner**
This PR adds an analysis tutorial which demonstrates how to load different output files of connectomemapper and how to analyze them.

### CODE REVIEW

File: notebooks/tutorial_environment.yml [hunk 1]
```diff
@@ -0,0 +1,251 @@
+name: cmp3-tutorial
```
Comment (sebastientourbier) [inline notebooks/tutorial_environment.yml:1]: Needs to review the list of packages. Only keep the packages that are imported in the notebooks. Do not forget also to add `- python=3.7` which is the python version used by CMP3. Finally, keep only the part specifying the version in the conda dependencies (i.e. `- nilearn=0.8.1`)

**Review by sebastientourbier — CHANGES_REQUESTED** [review]
Hi @mschoettner ! Please find the different points that should be corrected before we can be ready to merge your PR. In brief, it would consist of:

- [x] Reviewing the tutorial_environment.yml (following my comment).
- [x] Add CSV for all scales.

Then, on my side I will:

- [x] Integrate all CSV in cmtklib/data/... and implement a function that returns a numpy array containing the coordinates of each scale.
- [x] Review instructions in notebook
- [ ] Integrate the notebook in the docs and take care of moving /editing the part described in the README.
- [ ] Once VEPCON dataset is updated on OpenNeuro, we uncomment the lines necessary to clone it directly from there.

File: notebooks/tutorial_environment.yml (hunk 1 above)
Comment (mschoettner) [reply notebooks/tutorial_environment.yml:1]: Done, tested with the new environment

File: notebooks/tutorial_environment.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  - nibabel=3.2.1
+  - nilearn=0.8.1
+  - numpy=1.21.5
+  - pip=21.3.1
+  - scipy=1.7.3
+  - seaborn=0.11.0
+  - pip:
+    - pandas==1.3.5
+    - pybids==0.14.0
+    - pygsp==0.5.1
```
Comment (sebastientourbier) [inline notebooks/tutorial_environment.yml:22]: Add connectomemapper==3.0.3 right before merging

**Comment by sebastientourbier** [comment]
@mschoettner I should be done for now on my side. It would be great if you can check on your side if the notebook is still working properly! If yes, then the next and final steps would be to update the code related to the dataset and rerun the dataset. Then, when we are happy with the final results, I will integrate in the doc 😁

**Comment by sebastientourbier** [comment]
Preview of the doc available [here](https://connectome-mapper-3--141.org.readthedocs.build/en/141/) in prevision of the integration of a "read only" notebook.

**Pull request merged**